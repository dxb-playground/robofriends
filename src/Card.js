import React from 'react';

const Card = ({ id, name, email }) => {
  return (
    <div className='bg-light-green dib br3 pa3 ma2 grow'>
      <h1>RoboFriends</h1>
      <img alt='robots' src={`https://robohash.org/${id}?200x200`}/>
      <div>
        <h2>{name}</h2>
        <p>{email}</p>
      </div>
    </div>
  );
};

export default Card;
