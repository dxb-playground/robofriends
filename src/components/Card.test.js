import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Card from './Card';

it('expects to render the Card component', () => {
  let component = shallow(<Card />);

  expect(component.length).toEqual(1);
});
