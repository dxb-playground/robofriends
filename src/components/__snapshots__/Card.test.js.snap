// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render the Card component 1`] = `ShallowWrapper {}`;
